### PR TITLE
Close unused RuleTester instances in tests

### PR DESCRIPTION
**Pull request by sanjay990**

### CODE REVIEW

**Comment by sanjay990** [comment]
@amrutagokhale @arhimondr

File: presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestMergeAdjacentWindows.java [hunk 1]
```diff
@@ -41,10 +43,24 @@
 import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.window;
 import static com.facebook.presto.sql.tree.FrameBound.Type.CURRENT_ROW;
 import static com.facebook.presto.sql.tree.FrameBound.Type.UNBOUNDED_PRECEDING;
+import static io.airlift.testing.Closeables.closeAllRuntimeException;
 
 public class TestMergeAdjacentWindows
 {
-    private final RuleTester tester = new RuleTester();
```
Comment (findepi) [inline presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestMergeAdjacentWindows.java]: Searching for `private final RuleTester tester = new RuleTester();` yields 6 such places in `master`. Is this one special?

Comment (sanjay990) [inline presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestMergeAdjacentWindows.java]: No. It should be removed from all the files.

Comment (findepi) [inline presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestMergeAdjacentWindows.java]: OK. I renamed the PR accordingly.

**Comment by losipiuk** [comment]
Merged.

**Pull request merged**